api/controllers/game/draw.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (req, res) {
       ];
       const gameUpdates = {
         topCard: null,
-        log: [...game.log, userService.truncateEmail(user.email) + " drew a card"],
+        log: [...game.log, user.username + " drew a card"],
```

Comment:
It's only tangentially related (and not introduced by this change), but while we're here, why don't we start using ``-strings (template strings) for consistency and readibility:
```
 log: [...game.log, `${user.username} drew a card`],
```